buildSrc/src/main/kotlin/buildsrc/convention/publish-kotlin-multiplatform.gradle.kts

```diff
@@ -0,0 +1,56 @@
+package buildsrc.convention
+
+import buildsrc.config.createSkrapeItPom
+import gradle.kotlin.dsl.accessors._4ad077ad74816558e52d7069eb18a2f7.publishing
+import gradle.kotlin.dsl.accessors._4ad077ad74816558e52d7069eb18a2f7.signing
+import org.gradle.api.provider.Provider
+import org.gradle.api.publish.maven.MavenPublication
+import org.gradle.kotlin.dsl.create
+import org.gradle.kotlin.dsl.get
+import org.gradle.kotlin.dsl.maven
+import org.gradle.kotlin.dsl.provideDelegate
+
+plugins {
+    `maven-publish`
+    signing
+}
+
+description = "Configuration for publishing Kotlin Multiplatform libraries to Sonatype Maven Central"
+
+val signingKeyId: String? by project
+val signingKey: String? by project
+val signingPassword: String? by project
+
+val signingEnabled: Provider<Boolean> = provider {
+    signingKeyId != null && signingKey != null && signingPassword != null
+}
+
+val javadocJar by tasks.registering(Jar::class) {
+    group = JavaBasePlugin.DOCUMENTATION_GROUP
+    description = "Empty Javadoc Jar (required by Maven Central)"
+    archiveClassifier.set("javadoc")
+}
+
+publishing {
+    repositories {
+        // publish to local dir, for testing
+        maven(rootProject.layout.buildDirectory.dir("maven-internal")) {
+            name = "LocalProjectDir"
+        }
+    }
+    publications.withType<MavenPublication>().configureEach {
+        artifact(javadocJar)
+
+        createSkrapeItPom {
+            name.set("skrape{it} ${project.name}")
+        }
+    }
+}
```

Comment:
For some reason Sonatype isn't defined here.. 

I would expect to see something like this:

https://github.com/kotest/kotest/blob/945e029b4aea4ca39487f2dcf10d9a066dd7050a/buildSrc/src/main/kotlin/kotest-publishing-conventions.gradle.kts#L22-L31

The same in the `publish-jvm` convention plugin...

Was it removed, and it should be re-added now?